src/command.c

```diff
@@ -670,6 +670,14 @@ static bool cmd_shutdown_bmda(target_s *t, int argc, const char **argv)
 }
 #endif
 
+/*
+ * Heapinfo allows passing up to four int32_t from host to target.
```

Comment:
Reading this more closely, is it not 4 uint32_t's? They're not allowed to be negative, and they're stored firmware-side in target_addr_t which is an alias of uint32_t at this time.